sn_networking/src/lib.rs

```diff
@@ -158,9 +158,16 @@ pub struct Network {
     pub peer_id: Arc<PeerId>,
     pub root_dir_path: Arc<PathBuf>,
     keypair: Arc<Keypair>,
+    /// node owner's discord username, in readable format
+    owner: String,
```

Comment:
Can we store this `owner` field inside `struct Node`? The `Network` usually gets cloned a lot and we even avoid having `Arc<String>` here.

Reply:
I followed through the code and it seems like we only use this value during `quote` generation by the node. So `struct Node` would be a good fit for this. This would also avoid having the `onwer` values for `Client`